pom.xml

```diff
@@ -83,15 +83,16 @@
         <metrics.version>3.2.6</metrics.version>
         <aws.sdk.version>1.11.421</aws.sdk.version>
 
-        <specs2.version>4.3.2</specs2.version>
-        <junit.version>4.12</junit.version>
+        <specs2.version>4.9.2</specs2.version>
+        <junit.version>4.13</junit.version>
```

Comment:
@jeffklassen you may be interested to see that @elahrvivaz is bumping up the Specs 2 and Junit versions here.

Reply:
i think that i bumped this hoping to get some better stability in the tests - i didn't see any appreciable difference but seems like a good idea to use latest anyway.